File: LeetCode/2006差的绝对值为k的数对数目.cpp
int countKDifference(int* nums, int numsSize, int k){
    int cnt = 0;
    for (int i = 0; i < numsSize; i++) {
        for (int j = i + 1; j < numsSize; j++) {
            if (fabs(nums[i] - nums[j]) == k) cnt++;
        }
    }
    return cnt;
}

File: LeetCode/2006差的绝对值为K的数对数目.rb
# @param {Integer[]} nums
# @param {Integer} k
# @return {Integer}
def count_k_difference(nums, k)
    ans = 0
    (0...nums.size - 1).each do |x|
        (x...nums.size).each do |y|
            if (nums[x] - nums[y]).abs == k then
                ans += 1
            end
        end
    end
    return ans
end